packages/forma-36-react-components/tools/plop-templates/components/readme.mdx.hbs

```diff
@@ -1,23 +1,33 @@
 # {{pascalCase name}}
 
-A short summary of the component.
+* When to use the component, the challenges it can solve.
+* When to consider using another component (when there might be confusion).
+* (optional) What it consists of (if it's composed of other components).
 
-## Variations/types
+## Table of contents
 
-Provide the overview of the components variants.
+- [How to use {{pascalCase name}}](#how-to-use-{{pascalCase name}})
+- [Component variations](#component-variations)
+- [Code examples](#code-examples)
+- [Content recommendations](#content-recommendations)
+- [Accessibility](#accessibility)
 
-## Example of usage
+## How to use {{pascalCase name}}
```

Comment:
It's not totally clear to me how this section is different from the initial introduction. Could we have a short example maybe, or something?

Reply:
So maybe something from the recent CheckboxField example: this section contains info that checkboxes if possible should always be used with labels, that checkboxes should work independently or that they should have a logical order if used in the list. But also this is the place where we could give some recommendations for designers if there are any. It might be that for some components we would not need this section (some subcomponents for example). This is how I understand it.